lsp-pwsh.el

```diff
@@ -164,7 +168,7 @@ For more information about the brace styles please refer to https://github.com/P
   :package-version '(lsp-mode . "6.2"))
 
 (defcustom lsp-pwsh-developer-editor-services-log-level "Normal"
-  "Sets the logging verbosity level for the PowerShell Editor Services host executable.
+  "Sets the logging verbosity level for the PWSH Editor Services host exe.
```

Comment:
```suggestion
  "Sets the logging verbosity level for the PowerShell Editor Services host exe.
```

Reply:
Also, this sentence is from VsCode plugin customization, I think we should keep this as it.